homework_1/js/script.js

```diff
@@ -0,0 +1,5 @@
+document.addEventListener('DOMContentLoaded', () => {
+    let name = prompt('What is your name?','');
```

Comment:
This variant would be more correct, but we will review these questions in the closest future)
```suggestion
    const userName = prompt('What is your name?');
```